common/sgx/collateral.c

```diff
@@ -194,6 +194,40 @@ static oe_result_t _parse_sgx_extensions(
     return result;
 }
 
+static oe_result_t _hex_crl_to_byte(
+    uint8_t* crl_hex,
+    size_t crl_hex_size,
+    uint8_t** crl_byte_buffer,
+    size_t* crl_byte_buffer_size)
+{
+    oe_result_t result = OE_OK;
```

Comment:
Given this function is scoped to be used here, I think missing checks against inputs are fine. If later we want to make this shared by others, then we should add those checks.